### PR TITLE
Make Router include scripts in global space

### LINKED ISSUE #14: Router breaks globals
Because the scripts (or the user-provided router) is included inside a function, the variable globals that these scripts might be relying on setting and reading are no longer visible in the global space.

### PR DESCRIPTION
**Pull request by dontcallmedom**
fix #14

### CODE REVIEW

**Comment by nikkoss** [comment]
This patch should be updated a little.
There is a bug when a custom router script specified in codeception.yml is ignored.
The last line in the original code was "return Router::main();".
Though, it should be just "include Router::main();". So the custom router script gets executed.

**Comment by dontcallmedom** [comment]
Sorry, I'm not sure I understand what change you're asking to bring to the patch. Could you maybe use github in-line comments to make that clearer?

File: src/Codeception/Extension/Router.php [hunk 1]
```diff
@@ -45,4 +45,8 @@ static public function main()
     }
 }
 
-return Router::main();
+$res = Router::main();
+if ($res === false)
+  return $res;
```
Comment (tiger-seo) [inline src/Codeception/Extension/Router.php]: please, fix the styling
thanks

Comment (dontcallmedom) [inline src/Codeception/Extension/Router.php]: do you mean using {} for the if?

Comment (tiger-seo) [inline src/Codeception/Extension/Router.php]: yes, it must conform PSR-2

**Comment by bcole808** [comment]
This fix has helped me and I would love to see it merged into the project! I have been trying to serve WordPress with this extension and the namespace issue was causing problems with WordPress.

**Pull request merged**